### PR TITLE
docs: update zkevm-prover.md

### PR DESCRIPTION
**Pull request by joonkeep**
docs: update commit message format

### CODE REVIEW

**Comment by chokobole** [comment]
Could you please follow the convention [here](https://github.com/wemixkanvas/kanvas/blob/main/.gitmessage)?

**Comment by joonkeep** [comment]
docs: update commit message format

File: specs/zkevm-prover.md [hunk 1]
```diff
@@ -23,16 +23,16 @@
 
 ## Overview
 
-Prover produces a so called [ZK fault proof][g-zk-fault-proof] that states a [state][g-state] transition from `S` to
-`S'` is valid. It sounds like there's no big differences from validity proof. That's true. But the point is this is used
+The Prover produces a [ZK fault proof][g-zk-fault-proof] that states that a [state][g-state] transition from `S` to
```
Comment (chokobole) [inline specs/zkevm-prover.md:26]: Since Prover can be any one of several people, wouldn't it be more appropriate to use 'A' instead of 'The'?

```diff
-The Prover
+A Prover
```

Comment (joonkeep) [reply specs/zkevm-prover.md:26]: Good catch!

**Review by chokobole — APPROVED** [review]
LGTM

**Pull request merged**